vimrc

```diff
@@ -158,3 +158,6 @@ set diffopt+=vertical
 if filereadable($HOME . "/.vimrc.local")
   source ~/.vimrc.local
 endif
+
+set secure
```

Comment:
It might make sense to add in the future, but I think that it might be overkill for the initial implementation.
I would say that we can add it when we feel like it's an issue